discovery-infra/tests/test_logs.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import tarfile
+import time
+
+from assisted_service_client.rest import ApiException
+from tests.base_test import BaseTest
+
+class TestDownloadLogs(BaseTest):
+    def setup_hosts(self, cluster_id, api_client, node_controller):
```

Comment:
These are nice helper functions. They aren't only relevant for DownloadLogs scenarios. They can be very helpful as public helper functions

Reply:
yes, agree. you can move it to the base. And in general, if your using other methods here then "test_" so add a _ prefix to it, to differ from tests methods and make it more readable

Reply:
They will go into the code line with osher's PR and then I'll refactor mine code to use them